include/net/mpdccp_link_info.h

```diff
@@ -11,7 +11,8 @@
 struct mpdccp_link_info {
 
 	struct list_head	link_list;
-	char			name[IFNAMSIZ+1];
+//	char			name[IFNAMSIZ+1];
+	char			name[64];
```

Comment:
Why is IFNAMSIZ replaced by static 64 char?

Reply:
Because I chose to give the temporary virtual links a long name. The name consists of all the necessary information that is needed to identify what subflow it is for. A name could look like this: enp0s8_192.168.101.10:48092 - local interface, remote IP, and the remote port.
The names are bigger than 16 characters of IFNAMSIZ so I had to increase the string size in this case